### DISCUSSION TITLE
Multiple aggregation_key

### DISCUSSION THREAD

**Discussion by ngms17** [jertel/elastalert2#999] [Q&A] [answered]
Is it possible to set multiple `aggregation_keys`?

**Comment by jertel** [comment, marked as answer]
The code supports aggregation_key as a list type, but I haven't tried it:
```
aggregation_key:
  - field_one
  - field_two
```
If you have success using this syntax, would you mind submitting a PR with a brief note about it in the [docs](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#aggregation-key)?

**Reply by nsano-rururu** [reply]
https://github.com/Yelp/elastalert/issues/1190
https://github.com/Yelp/elastalert/issues/1568
